registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+  {
+    "ticker": "DD666",
+    "poolId": "9cb4471bad9979623d32ad3c14ff14b340f046e2a5b3ae2e3c8f575b",
+    "social": {
+      "website": "https://devilsdelegate.com",
+      "twitter": "DevilsDelegate",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "DevilsDelegate",
```

Comment:
Please can you add a user's TG handle? This must be the handle of a user and not a group or channel.